### PR TITLE
PR for review

### PR DESCRIPTION
**Pull request by visva-dev**
## Weather App
- The Weather App project comes at the end of the Asynchronous Javascript and APIs section of the Javascript course.
- It tests the knowledge of the student about asynchronous communication with promises or async/await and when to use them.
- Users can get weather information using API.

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hello @visva-dev  👋

**You've done quite on this. 💪**

However, let's make a few changes.

- [x]  Please add a toggle button so the user would be able to toggle between Celsius and Fahrenheit temperature states.

- [x] OPTIONAL - You can add a loading animation to display before the data loads.

- [x]  Kindly refer to other comments on your code.

### **STATUS: CHANGES REQUESTED** 👷‍♂️

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

HAPPY CODING👨‍💻

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+  hilow.innerText = `${Math.round(weather.main.temp_min)}°c / ${Math.round(
+    weather.main.temp_max,
+  )}°c`;
+}
+
+function getResults(query) {
+  fetch(`${url.base}weather?q=${query}&units=metric&APPID=${process.env.API}`)
+    .then((weather) => { weather.json().then(displayResults); });
+}
+
```
Comment (mikenath223) [inline src/index.js]: - [x] Please break up this code. You shouldn't dump all your code in `index.js`. **Separate logic code from DOM manipulation code into different modules.** Kindly do that.

Comment (mikenath223) [inline src/index.js]: - [x] You should also have a way to handle errors from the fetch request perhaps by chaining a `catch`  or using `try/catch block`.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "Weather-App",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
+  },
```
Comment (mikenath223) [inline package.json]: - [x] Kindly add other necessary scripts in this section:
```
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },

```

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+      'April',
+      'May',
+      'June',
+      'July',
+      'August',
+      'September',
+      'October',
+      'November',
+      'December',
+    ];
```
Comment (mikenath223) [inline src/index.js]: - [x] Also please use ES6 arrow function syntax. **Fix this and other similar functions.**

**Review by rOluochKe — APPROVED** [review]
Hello @visva-dev 
### You got it right, well done. The project has been approved. Congratulations :boom: :100:
### Merge and submit the project. Good luck with the next one.
- Happy coding!

**Pull request merged**